app/services/show_attendee_phone_number_for_user.rb

```diff
@@ -0,0 +1,16 @@
+class ShowAttendeePhoneNumberForUser
+    def initialize(current_user, attendance)
+        @current_user = current_user
+        @attendance = attendance
+    end
+
+    def call
+        # logic operating on @current_user and @attendance
+        # returns true/false
+        @current_user.host? &&
+            @attendance.todo? &&
```

Comment:
I've recently adopted this pattern that I really like for methods that return bool:

```ruby
def whatever
  return false if some_condition
  return false if !some_condition && some_other_condition
  return true
end
```

This gives you the single line by line and a set of assertions that you can translate into unit tests. Also makes for very readable code that lets you step through and reason about what's being returned.